### PR TITLE
Adds sidekiq batching & actioncable for live reloading posts

### PR DESCRIPTION
**Pull request by dennyluan**
#### Sidekiq batching & actioncable for live reloading page on parse job completion

This PR adds live reloading for new posts using the`UploadParsePipelineWorker` which runs everything in background jobs and then sends a websocket message to the post view to re-fetch the post data. I've consolidated some of the post view into the `PostEditor` component as part of this.

The general flow follows something like this.

- [x] 1. User create upload => controller redirects to posts/:id, freeze post with loading icon
- [x] 2. Callback chain
  - Upload.create!
    - create_upload_tei =>
      - GrobidServiceWorker.perform_async(id) =>
        - DiborgServiceWorker.perform_async(@upload_tei.id, @upload_tei.post.id)
        - FiguresExtractWorker.perform_async(id) =>
          - FiguresFetcherWorker.perform_async(figures_response, @upload.id)
- [x] 3. batch.on(:success) sends message to subscribers `posts_#{id}_channel`
- [x] 4. rerender component

Also this adds a little blue favicon for in local dev mode so you can tell tabs apart.

### CODE REVIEW

**Comment by dennyluan** [comment]
As is tradition, no tests :D

File: app/javascript/components/PostEditor.js [hunk 1]
```diff
@@ -37,23 +39,50 @@ class PostEditor extends React.Component {
...
 	}
 
+  componentDidMount(){
+    var post = document.getElementById('post').getAttribute("data-post-id")
+    var cableHost = this.state.post.data.attributes.cable_url
+    var cable = createConsumer(cableHost)
+
+    cable.subscriptions.create({channel: "PostsChannel", post_id: post}, {
+      connected() {
+        console.log("connected to PostsChannel")
```
Comment (cindywu) [inline app/javascript/components/PostEditor.js]: there are 3 console.log in here. are these left here intentionally?

File: app/javascript/channels/posts_channel.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// import consumer from "./consumer"
```
Comment (cindywu) [inline app/javascript/channels/posts_channel.js]: we have an entire file here that is commented out. why?

Comment (dennyluan) [reply app/javascript/channels/posts_channel.js]: thanks, ill remove this

File: app/javascript/components/PostEditor.js (hunk 1 above)
Comment (dennyluan) [reply app/javascript/components/PostEditor.js]: Removed two of them.

**Pull request merged**